packages/pirania/files/usr/bin/captive-portal

```diff
@@ -62,31 +83,40 @@ set_iptables () {
 			$iptables -t $table -N pirania
 		done
 
-		# send the packets that we want to filter to the pirania chains
-		# 'nat' table is consulted when a packet that creates a new connection is encountered and with PREROUTING, as soon as they come in.
-		$iptables -t nat -$AorI PREROUTING --jump pirania
-		# 'filter' table is the default table, FORWARD for packets being routed through the box
-		$iptables -t filter -$AorI FORWARD --jump pirania
-
-		# we are going to modify only the packets specified in 'catch_interfaces', so sending them to the pirania chain
-		for interface in $(uci get pirania.base_config.catch_interfaces); do
-			$iptables -t mangle -$AorI PREROUTING -i $interface -j pirania
-		done
+		# Redirect to pirania chain every packet from catch_wifi_mode_ifaces
+		if [ $catch_wifi_mode_ifaces = "yes" ] ; then
```

Comment:
here you could check instead if the list is not empty (the "" quotes are important, yes, bash is horrible)
`if [ -n "$(uci get pirania.base_config.wifi_mode_ifaces_catched)" ] ; then`